clang.rst

```diff
@@ -128,10 +132,10 @@ The installer does not install all the components needed on occasion. For
 example, you might want to run a ``scan-build`` or examine the results with
 ``scan-view``. You can copy the components by hand with: ::
 
-    sudo mkdir /usr/local/bin/scan-build
-    sudo cp -r llvm-3.4/tools/clang/tools/scan-build /usr/local/bin
-    sudo mkdir /usr/local/bin/scan-view
-    sudo cp -r llvm-3.4/tools/clang/tools/scan-view /usr/local/bin
+   $ sudo mkdir /usr/local/bin/scan-build
+   $ sudo cp -r llvm-3.4/tools/clang/tools/scan-build /usr/local/bin
+   $ sudo mkdir /usr/local/bin/scan-view
+   $ sudo cp -r llvm-3.4/tools/clang/tools/scan-view /usr/local/bin
```

Comment:
Keep in mind that in many of these examples, the ``$`` are intentionally omitted to make the commands easy to copy-paste.  Does the highlight not work without the ``$``?

Reply:
Without the $, the `console` lexer treats it as output. A workable alternative, is to omit the prompt and choose the `bash` lexer as I did adjacently. In this case,  after $, no highlighting is generated anyway so `bash`, `none`, `console`: it's all one!